.gitlab-ci.yml

```diff
@@ -267,30 +267,6 @@ linux_armhf_go_deps:
   mkdir vendor && tar xzf vendor.tar.gz -C vendor
   rm -f go-bin.tar.gz go-pkg.tar.gz vendor.tar.gz
 
-# run tests for windows-64
```

Comment:
Why do we remove those ?

Reply:
They were here just because we were not able to run the tests on Windows with Go < 1.14 with the `-race` flag.
Now that we can, we don't need those anymore because they covered by the "regular" Windows tests.